geometry/mesh.hpp

```diff
@@ -230,6 +230,10 @@ struct Mesh {
     /** Calculate face barycenter.
      */
     math::Point3 barycenter(const Face &face) const;
+
+    /** Calculate volume of a mesh.
```

Comment:
I'd add a note that the formula only works on watertight meshes. (may seem obvious, but...)

```suggestion
    /** Calculate volume of a watertight mesh.
```